hedera-mirror-rest/topicmessage.js

```diff
@@ -158,16 +167,113 @@ const getTopicMessages = async (req, res) => {
   res.locals[constants.responseDataLabel] = topicMessagesResponse;
 };
 
-const extractSqlFromTopicMessagesRequest = (topicId, filters) => {
+async function getSequenceNumberFromTopicMessageLookup(topicId, order, limit) {
```

Comment:
follow the existing coding style
```suggestion
const getSequenceNumberFromTopicMessageLookup = async (topicId, order, limit) {
```